Dockerfile

```diff
@@ -9,14 +9,19 @@ ARG TAGS="sqlite sqlite_unlock_notify"
 ENV TAGS "bindata timetzdata $TAGS"
 ARG CGO_EXTRA_CFLAGS
 
-#Build deps
-RUN apk --no-cache add build-base git nodejs npm
+# Build deps
+RUN apk --no-cache add \
+    build-base \
+    git \
+    nodejs \
+    npm \
+    && rm -rf /var/cache/apk/*
```

Comment:
Given there is already `--no-cache`, is this `rm` really needed?

Reply:
It depends. Because we always inherit some base image and we don't know state of his cache directory. Therefore, in my opinion, it is worth leaving this command.